rpc/src/main/scala/server/GrpcConfig.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.rpc
+package server
+
+import java.io.File
+import java.util.concurrent.Executor
+
+import io.grpc._
+
+case class Config(port: Int)
+
+sealed trait GrpcConfig
```

Comment:
We should probably add `extends Product with Serializable`, otherwise the type of something like `List(GetDirectExecutor)` would be inferred as `List[GrpcConfig with Product with Serializable]`.